vcpkg.json

```diff
@@ -31,20 +31,28 @@
             "wayland"
           ],
           "platform": "linux"
-        }
+        },
+        "curl",
+        {
+          "name": "libarchive",
+          "default-features": false
+        },
+        "nlohmann-json"
       ]
     },
     "flatpak-libs": {
       "description": "System libraries missing in the Flatpak runtime.",
       "dependencies": [
         "libmad",
-        "glew"
+        "glew",
+        "nlohmann-json"
       ]
     },
     "steam-libs": {
       "description": "System libraries not provided in the Steam Sniper Runtime.",
       "dependencies": [
-        "libmad"
+        "libmad",
+        "nlohmann-json"
```

Comment:
Add  libarchive here too, cause it's missing from Steam

Reply:
yup